data/pages/faq-grantees.mdx

```diff
@@ -127,22 +127,38 @@ publish them later, is not compliant with our grant agreement.
 
 ### How does OpenSats evaluate the work done by a grantee?
 
-Every grantee must submit a progress report every 90 days. A markdown template
-is provided when we request your first report. The report in markdown format must be sent to support@opensats.org.
+All grantees must submit a progress report every 90 days at
+<https://opensats.org/reports/submit>
 
-If your grant started in March, for example, the 1st report will be due anytime
-during the month of June and should cover the work done from March through May,
-and so on and so forth. You have a 30-day window to submit the report and we
-appreciate if you submit them on time.
+**First-time grantees** must also submit 30-day progress reports during the
```

Comment:
Perhaps clarify if first-time at OpenSats (grantees could be moving to OpenSats after bitcoin (or nostr?) open source grants from other funding orgs).

```suggestion
**First-time OpenSats grantees** must also submit 30-day progress reports during the
```